src/libaktualizr/storage/sql_utils.h

```diff
@@ -8,6 +8,10 @@
 
 #include "logging/logging.h"
 
+extern int schema_migrations_len;
```

Comment:
Using a vector is nice! We could also get rid of the `_len` field and just call `.size()`.
`current_schema` could be declared as a `std::string` as well, for consistency.

Also minor but did you have a specific reason to put this declarations in `sql_utils.h`? I think they could simply live at the top of `sqlstorage.cc` with a comment explaining where they are coming from.